src/parser/jobs/ast/modules/Sect/Sect.tsx

```diff
@@ -197,42 +124,22 @@ export default class Sect extends Module {
 		}
 
 		// Statistic box
-		const icon = !this.activeSectId ? NO_SECT_ICON
-			: ACTIONS.DIURNAL_SECT.id === this.activeSectId ? ACTIONS.DIURNAL_SECT.icon
-				: ACTIONS.NOCTURNAL_SECT.icon
+		const icon = this.activeSectId === 0 ? noSectIcon
+			: this.data.actions.DIURNAL_SECT.id === this.activeSectId ? this.data.actions.DIURNAL_SECT.icon
+				: this.data.actions.NOCTURNAL_SECT.icon
 		const noSectValue = <Trans id="ast.sect.info.no-sect-detected">No sect detected</Trans>
-		const value = !this.activeSectId ? noSectValue :
-			ACTIONS.DIURNAL_SECT.id === this.activeSectId ? ACTIONS.DIURNAL_SECT.name
-				: ACTIONS.NOCTURNAL_SECT.name
+		const value = this.activeSectId === 0 ? noSectValue
+			: this.data.actions.DIURNAL_SECT.id === this.activeSectId ? this.data.actions.DIURNAL_SECT.name
+				: this.data.actions.NOCTURNAL_SECT.name
```

Comment:
Consider checking explicitly for the 2 sects and defaulting to no sect at the end as your fallback.

Reply:
reordered per suggestion